source/plugins/vxlan.rst

```diff
@@ -328,70 +234,134 @@ Since VXLAN uses UDP packet to forward encapsulated the L2 frames,
 UDP/8472 port must be opened.
 
 
-Configure in RHEL or CentOS
-'''''''''''''''''''''''''''
-
-RHEL and CentOS use iptables for firewalling the system, you can open
-extra ports by executing the following iptable commands:
+Make sure that your firewall (firewalld, ufw, ...) allows UDP packets on port 8472, as an example:
 
 ::
 
-   $ sudo iptables -I INPUT -p udp -m udp --dport 8472 -j ACCEPT
-
-
-These iptable settings are not persistent accross reboots, we have to
-save them first.
-
-::
-
-   $ sudo iptables-save > /etc/sysconfig/iptables
-
-
-With this configuration you should be able to restart the Network,
-although a reboot is recommended to see if everything works properly.
-
-::
-
-   $ sudo service network restart
-   $ sudo reboot
+   $ sudo firewall-cmd --zone=public --permanent --add-port=8472/udp
+   $ sudo ufw allow proto udp from any to any port 8472
 
 
-.. warning::
-   Make sure you have an alternative way like IPMI or ILO to reach the machine
-   in case you made a configuration error and the Network stops functioning!
 
+VXLAN using EVPN
+---------------------
+Using VXLAN with BGP+EVPN as underlay is more complex to set up, but does allow for more scaling and provides much more flexibility.
 
-Configure in Ubuntu
-'''''''''''''''''''
+This documentation can not cover all elements of deploying BGP+EVPN in your environment.
 
-The default firewall under Ubuntu is UFW (Uncomplicated FireWall), which
-is a Python wrapper around iptables.
+It is recommend to read `this blogpost <https://vincent.bernat.ch/en/blog/2017-vxlan-bgp-evpn>`_ before you continue. 
 
-To open the required ports, execute the following commands:
+The main items for using EVPN:
 
-::
+- BGP Routing Daemon on the hypervisor
+- No LACP/Bonding will be used
+- Modified script is required to use EVPN instead of Multicast
```

Comment:
```suggestion
- The modified script is required
```

Reply:
@NuxRo Did you truncate the phrase by accident?